src/app/landing/landing-hero/landing-hero.component.ts

```diff
@@ -1,14 +1,22 @@
-import { Component, Output, EventEmitter } from '@angular/core';
+import { Component } from '@angular/core';
+import { MatDialog } from '@angular/material/dialog';
+import { DialogsVideoComponent } from '../../shared/dialogs/dialogs-video.component';
 
 @Component({
   selector: 'planet-landing-hero',
   templateUrl: 'landing-hero.component.html',
   styleUrls: [ 'landing-hero.scss' ]
 })
 export class LandingHeroComponent {
-  @Output() playVideoEvent = new EventEmitter<boolean>();
+  constructor(
+    private dialog: MatDialog
+  ) {}
 
   playVideo() {
-    this.playVideoEvent.emit(true);
+    this.dialog.open(DialogsVideoComponent, {
+      data: {
+        videoUrl: 'assets/landing-page/video/landing.mp4'
```

Comment:
Right now I do not see this PR improving any implementation. @dogi what is the plan for this reason to be removed from image?

Reply:
In the current state the emitted event is not being used anywhere and the video is not playing on click.

Reply:
Need to test both container and non container case due to relative path. So please test with different language, before merging.